pages_builder/pages/search-result.yml

```diff
@@ -0,0 +1,96 @@
+pageTitle: Notification banners - Test toolkit
+assetPath: govuk_template/assets/
+head: >
+  <link rel="stylesheet" media="all" type="text/css" href="public/stylesheets/index.css" />
+  <link rel="stylesheet" media="all" type="text/css" href="public/stylesheets/grids/index.css" />
+  <link rel="stylesheet" media="all" type="text/css" href="public/stylesheets/_search-result-wrapper.css" />
+content: >
+  <div id="global-breadcrumb" class="header-context">
+    <nav>
+      <ol class="group" role="breadcrumbs">
+        <li>
+          <a href="./">Home</a>
+        </li>
+      </ol>
+    </nav>
+  </div>
+  <main role="main" id="content" class="wrapper">
+    <div id="wrapper">
+      <header class="page-heading">
+        <h1>Search result</h1>
+      </header>
+      <div class="grid-row">
+        <div class="two-third-column">
+          <div class="search-result">
+            <h1 class="search-result-title">
+              <a href="#">Digital Marketplace frontend toolkit as a service</a>
+            </h1>
+            <p class="search-result-supplier">
+              by Knight Technology Ltd.
+            </p>
+            <p class="search-result-excerpt">
+              Example markup can be found in the
+              <span class="search-result-highlighted-text">pages folder</span>.
+              It is used to generate the example pages for documentation into the
+              pages folder. Generating the documentation pages. The
+              documentation pages can be generated by following these steps.
+            </p>
+            <ul class="search-result-tags">
```

Comment:
Seems a little odd that the fact this list isn't identified as tags. Can we bake this information into it?
This could be done by adding a visually hidden heading or by using the `aria-label` to give them a label of 'Tags'.